staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation_test.go

```diff
@@ -25,6 +25,11 @@ import (
 	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
 )
 
+const (
+	costLimit   = 100000000
+	intOrString = "int-or-string"
```

Comment:
consistency nit: inline intOrString like done with "duration", "date", "date-time", or alternatively, make constants for all of them?

Reply:
Inline sounds good to me - I'll resolve this once I push a commit.